javascript/javascript1/week1/answers.js

```diff
@@ -0,0 +1,83 @@
+/* 
+//************************************************ 
+STEP2 answer
+//************************************************ 
+
+Alejandro Gispert
+https://www.freecodecamp.org/AlejandroGispert
+
+*/
+
+//************************************************ 
+//Age-ify (A future age calculator) 
+//************************************************ 
+
+const yearOfBirth = 1986
+const yearFuture = 2045
+
+const age = yearFuture - yearOfBirth
+
+console.log("You will be " + age +" years old in " + yearFuture)
+
+//************************************************ 
+//Goodboy-Oldboy (A dog age calculator)
+//************************************************ 
+
+
+const dogYearOfBirth = 2010
+const dogYearFuture = 2045
```

Comment:
nitty-picky: Are those the correct years from the assignment?

Reply:
done